docs/pages/guides/spend-permissions/quick-start.mdx

```diff
@@ -659,23 +576,23 @@ This code is using our spender client to do two things:
 1. calls `approveWithSignature` to approve the spend permission
 2. calls `spend` to make use of our allowance and spend our user's funds
 
-Since our spender is a smart contract wallet, notice that these calls are formulated as userOperations instead of 
-direct transactions. They'll be submitted to the blockchain by a bundler and the gas will be subsidized by our
-Coinbase Developer Platform paymaster.
 
 ### Try out your app
 
 Run your app locally with `bun run dev` and visit `localhost:3000`.
 
-You should see a "Connect wallet" button in the top right corner; click that.
+When you click the "Subscribe" button you should be prompted to create or connect your Smart Wallet.
 
 You can create a new Smart Wallet via the popup. Note that you'll need a little ETH in this wallet to fund the
 deployment of your account. If you don't have any testnet ETH, try this [Coinbase faucet](https://portal.cdp.coinbase.com/products/faucet).
 
-Once your wallet is created and funded, return to the app and click "Subscribe".
-Sign the prompts, which will deploy your new Smart Wallet (if undeployed), and then prompt you to sign over the spend permission.
+Before you continue, note that you'll also need a little testnet ETH in your spender wallet, which will need to fund the gas for the transactions.
+
+Once your wallet is created and both wallets are funded, return to the app and click "Subscribe".
+Sign the prompts, which will deploy your new Smart Wallet (if undeployed), and then prompt you to sign the spend permission requested by the app.
```

Comment:
I don't think we prompt to deploy here - and if we did for base sepolia the default paymaster would handle funds for us

Reply:
got it -- yeah as I mentioned in my comment above, I don't see this when I just tried it fresh so will remove